test/test_manual.jl

```diff
@@ -361,15 +361,42 @@ isdir(outdir) && rm(outdir, recursive=true)
     @test LatticeBoltzmannEquations2D(Ma=nothing, Re=nothing, u0=1, nu=1) isa LatticeBoltzmannEquations2D
   end
 
-  @testset "LBM functions" begin
+  @testset "LBM 3D constructor" begin
+    # Neither Mach number nor velocity set
+    @test_throws ErrorException LatticeBoltzmannEquations3D(Ma=nothing, Re=1000)
+    # Both Mach number and velocity set
+    @test_throws ErrorException LatticeBoltzmannEquations3D(Ma=0.1, Re=1000, u0=1)
+    # Neither Reynolds number nor viscosity set
+    @test_throws ErrorException LatticeBoltzmannEquations3D(Ma=0.1, Re=nothing)
+    # Both Reynolds number and viscosity set
+    @test_throws ErrorException LatticeBoltzmannEquations3D(Ma=0.1, Re=1000, nu=1)
+
+    # No non-dimensional values set
+    @test LatticeBoltzmannEquations3D(Ma=nothing, Re=nothing, u0=1, nu=1) isa LatticeBoltzmannEquations3D
+  end
+
+  @testset "LBM 2D functions" begin
     # Set up LBM struct and dummy distribution
     equation = LatticeBoltzmannEquations2D(Ma=0.1, Re=1000)
     u = Trixi.equilibrium_distribution(1, 2, 3, equation)
 
     # Component-wise velocity
     @test isapprox(Trixi.velocity(u, 1, equation), 2)
     @test isapprox(Trixi.velocity(u, 2, equation), 3)
+  end
+
+  @testset "LBM 3D functions" begin
+    # Set up LBM struct and dummy distribution
+    equation = LatticeBoltzmannEquations3D(Ma=0.1, Re=1000)
+    u = Trixi.equilibrium_distribution(1, 2, 3, 4, equation)
+
+    # Component-wise velocity
+    @test isapprox(Trixi.velocity(u, 1, equation), 2)
+    @test isapprox(Trixi.velocity(u, 2, equation), 3)
+    @test isapprox(Trixi.velocity(u, 3, equation), 4)
```

Comment:
```suggestion
    @test isapprox(velocity(u, 1, equation), 2)
    @test isapprox(velocity(u, 2, equation), 3)
    @test isapprox(velocity(u, 3, equation), 4)
```
We export `velocity`, don't we?

Reply:
You're right!